src/v/redpanda/admin/api-doc/broker.json

```diff
@@ -310,6 +327,71 @@
                     "description": "failed transfer partition count"
                 }
             }
+        },
+        "decommission_status": {
+            "id": "decommission_status",
+            "description": "Node decommissioning status",
+            "properties": {
+                "finished": {
+                    "type": "boolean",
+                    "description": "indicate if decommissioning is finished"
+                },
+                "replicas_left": {
+                    "type": "long",
+                    "description": "number of replicas left on a node"
+                },
+                "partitions": {
+                    "type": "array",
+                    "items": {
+                        "type": "partition_reconfiguration_status"
+                    },
+                    "description": "Array of partition reconfiguration statues"
+                }
+            }
+        },
+        "partition_reconfiguration_status": {
+            "id": "partition_reconfiguration_status",
+            "description": "Partition reconfiguration status",
+            "properties": {
+                "ns": {
+                    "type": "string",
+                    "description": "namespace"
+                },
+                "topic": {
+                    "type": "string",
+                    "description": "topic"
+                },
+                "partition": {
+                    "type": "int",
+                    "description": "partition"
+                },
+                "moving_to": {
+                    "type": "broker_shard",
+                    "description": "information where the partition is being moved"
+                },
+                "bytes_left_to_move": {
+                    "type": "long",
+                    "description": "bytes left to move to new replica"
+                },
+                "target_size": {
```

Comment:
nit: consider using a "total_bytes_to_move" analogous to "bytes_left_to_move".. I think "target" will be a bit vague to the operator. because there are two parties here.

Reply:
good point, i will include current partition size and amount of data that were transfer to target node